util/hb-info.cc

```diff
@@ -50,8 +50,8 @@ struct info_t
       {"show-glyph-count",0, 0, G_OPTION_ARG_NONE,	&this->show_glyph_count,	"Show glyph count",		nullptr},
 
       {"list-all",	0, 0, G_OPTION_ARG_NONE,	&this->list_all,		"List all long information",	nullptr},
-      {"list-names",	'l', 0, G_OPTION_ARG_NONE,	&this->list_names,		"List names",			nullptr},
-      {"list-tables",	'l', 0, G_OPTION_ARG_NONE,	&this->list_tables,		"List tables",			nullptr},
+      {"list-names",	0, 0, G_OPTION_ARG_NONE,	&this->list_names,		"List names",			nullptr},
+      {"list-tables",	0, 0, G_OPTION_ARG_NONE,	&this->list_tables,		"List tables",			nullptr},
```

Comment:
I thought you want to keep it for `--list-tables`

Reply:
Err. RIght. Sorry.